<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicSoundKMicroWidget</class>
 <widget class="QWidget" name="MusicSoundKMicroWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择扫描范围</string>
  </property>
  <widget class="QWidget" name="topWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QLabel" name="topTitleName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>220</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
    </property>
    <property name="text">
     <string>一起来K歌</string>
    </property>
   </widget>
   <widget class="QToolButton" name="topTitleCloseButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="controlWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>660</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="MusicMovingLabelSlider" name="timeSlider">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>660</width>
      <height>15</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>25</y>
      <width>80</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00/00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="stateButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>35</y>
     <width>660</width>
     <height>405</height>
    </rect>
   </property>
   <widget class="QWidget" name="videoPage"/>
   <widget class="QWidget" name="musicPage"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicMovingLabelSlider</class>
   <extends>QSlider</extends>
   <header>musicmovinglabelslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
